launch-game.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+TEMPLATE_LAUNCHER=$(python -c "import os; print(os.path.realpath('$0'))")
+TEMPLATE_ROOT=$(dirname "${TEMPLATE_LAUNCHER}")
+source ${TEMPLATE_ROOT}/mod.config
+
+MOD_SEARCH_PATHS="./../.."

```

Comment:
It's weird - blame definitely retains the history. https://github.com/OpenRA/OpenRA/blame/bleed/OpenRA.Mods.Cnc/Traits/Chronoshiftable.cs